### PR TITLE
fix: Add storeAsInitialCamera parameter to StackViewport setCamera

### PR DESCRIPTION
**Pull request by volrath**
While reading the API documentation on [`StackViewport.setZoom`](https://www.cornerstonejs.org/api/core/class/StackViewport#setZoom) I wanted to try out the `storeAsInitialCamera` parameter, but I noticed it wasn't working. Upon further inspection I realized it was due to the fact that, even though `Viewport.setCamera` has said parameter, `StackViewport` doesn't, so the the value is never passed down to the super class.

Please note that I purposely omitted the parameter from the call to `this.setCameraCPU` given that I believe it is not used anywhere. Not sure if this is the correct take though, so let me know if this requires further work.

### CODE REVIEW

**Comment by wayfarer3130** [comment]
This is a simple fix, looks correct to me.  @sedghi wasnt sure if I should approve this one, or if you wanted a peak first.

**Review by sedghi — APPROVED** [review]

**Comment by sedghi** [comment]
Thanks @volrath , this is a great PR

**Pull request merged**
